daemon/daemon.go

```diff
@@ -827,7 +827,7 @@ func (d *Daemon) init() error {
 
 		// Set up the list of IPCache listeners in the daemon, to be
 		// used by syncLXCMap().
-		d.ipcacheListeners = []ipcache.IPIdentityMappingListener{d, &envoy.NetworkPolicyHostsCache}
+		d.ipcacheListeners = []ipcache.IPIdentityMappingListener{&envoy.NetworkPolicyHostsCache, d}
```

Comment:
Is this a functional change? If so, would it be helpful to update the comment to describe why it's in this order?

Reply:
Yes it's a functional change.
Configuring xDS first will likely make Envoy get configured earlier, so it'll be ready earlier when policies change.

Reply:
Updated the commit message.